packages/contracts-bedrock/scripts/DeployAuthSystem.s.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Script } from "forge-std/Script.sol";
+import { CommonBase } from "forge-std/Base.sol";
+import { stdToml } from "forge-std/StdToml.sol";
+
+import { GnosisSafe as Safe } from "safe-contracts/GnosisSafe.sol";
+
+import { DeployUtils } from "scripts/libraries/DeployUtils.sol";
+import { Solarray } from "scripts/libraries/Solarray.sol";
+
+contract DeployAuthSystemInput is CommonBase {
+    using stdToml for string;
+
+    // Generic safe inputs
+    // Note: these will need to be replaced with settings specific to the different Safes in the system.
+    uint256 internal _threshold;
+    address[] internal _owners;
+
+    function set(bytes4 _sel, uint256 _value) public {
+        if (_sel == this.threshold.selector) _threshold = _value;
+        else revert("DeployAuthSystemInput: unknown selector");
+    }
+
+    function set(bytes4 _sel, address[] memory _addrs) public {
+        if (_sel == this.owners.selector) {
+            for (uint256 i = 0; i < _addrs.length; i++) {
+                _owners.push(_addrs[i]);
```

Comment:
Before the for loop we should have `require(_owners.length == 0, "DeployAuthSystemInput: already set")`, since calling this twice appends to the array instead of overwriting it

Reply:
Goo callout, will do